docs/cpp/source/check-doxygen.sh

```diff
@@ -44,9 +46,12 @@ ignore_warning "warning: explicit link request to 'Item' could not be resolved"
 # Count the number of remaining warnings.
 warnings="$(grep 'warning:' doxygen-log.txt | wc -l)"
 
+echo "Treating all remaining warnings as errors"
```

Comment:
if there are no other warnings from the doxygen build, will "all remaining warnings are errors" message be confusing if the next output is warnings from subsequent build steps?

Reply:
I don't see how that could happen; as per this script, the only thing that is printed in the rest of this script is `doxygen-log.txt`, which is exactly the file we counted the number of warnings from. Am I misunderstanding your comment, perhaps?

Reply:
I may be misinterpreting what this does or when it's run, I don't actually know what doxygen is. I'm just thinking I'm going to look at some CI output and see "treating all remaining warnings as errors" and then if there aren't any warnings, on the next line see some warning from jenkins or pip or from a unit test or whatever the next unrelated step is in the build.

Reply:
It looks like this: https://travis-ci.org/pytorch/pytorch/jobs/530755597

I think it will be hard to get confused about the output in this situation. There are no further jenkins, pip or unit tests run in this job.

Reply:
yeah that's clear enough, thanks